lib/std/math/big/int.zig

```diff
@@ -19,6 +19,28 @@ const native_endian = builtin.cpu.arch.endian();
 
 const debug_safety = false;
 
+// SAFETY: the caller must ensure it won't overflow a Limb
```

Comment:
1. What does "it" mean? Please be more explicit.
2. Style guide says to use "assume" and "assert" https://ziglang.org/documentation/master/#toc-Doc-Comment-Guidance